simpinkscr/simple_inkscape_scripting.py

```diff
@@ -2093,12 +2093,16 @@ def effect(self):
             code += '\n'
         if self.options.program is not None:
             code += self.options.program.replace(r'\n', '\n')
+        #remove unnecessary import
+        code.replace("from simpinkscr import *", "")
         try:
             exec(code, sis_globals)
         except SystemExit:
             pass
         _simple_top.replace_all_guides(sis_globals['guides'])
 
+def main():
+    SimpleInkscapeScripting().run()
```

Comment:
Is this required for VSCode integration?  If not, why define a `main` function here and in `svg_to_simp_ink_script.py`?